org_fedora_hello_world/dbus_service/hello_world_interface.py

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+from dasbus.server.interface import dbus_interface
+from dasbus.server.property import emits_properties_changed
+from dasbus.typing import *  # pylint: disable=wildcard-import
+
+from pyanaconda.modules.common.base import KickstartModuleInterface
+
+from org_fedora_hello_world.constants import HELLO_WORLD
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+@dbus_interface(HELLO_WORLD.interface_name)
+class HelloWorldInterface(KickstartModuleInterface):
+    """The interface for HelloWorld.
+
+    The interface class is needed for interfacing code running within
+    Anaconda's main process and code running in the service process. The dasbus
```

Comment:
I recommend:

"in the **DBus** service process."

Reply:
I'm not sure about the right terminology here - will look into this.

Reply:
Ah, looks like it is indeed a "D-Bus service" in the freedesktop docs. Changed accordingly.